docs/dev/perl5/news/2013/perl-5.18.1.html

```diff
@@ -0,0 +1,75 @@
+[% page.title = "Perl 5.18.1 Release Announcement" %]
+<meta charset="utf-8">
+
+<p>
+We are happy to announce perl 5.18.1 the second stable release of version
+18 of Perl 5.
+</p>
+
+<p>
+You will soon be able to download Perl 5.18.1 from your favorite CPAN
+mirror or find it at:
+</p>
+
+<p>
+https://metacpan.org/release/RJBS/perl-5.18.1/
+</p>
+
+<p>
+SHA1 digests for this release are:
+</p>
+
+<p>
```

Comment:
I was trying to more or less match the style of the most recent page in this directory: http://dev.perl.org/perl5/news/2012/perl-5.16.2.html